SPECS/python-more-itertools/python-more-itertools.spec

```diff
@@ -37,13 +36,17 @@ Summary:        %{summary}
 %pyproject_save_files more_itertools
 
 %check
-%tox
+pip3 install tox
```

Comment:
Would it be possible to use our own `python3-tox` package instead of using an external version from pip? We allow network access during tests, but that's mostly a workaround, since we don't have test dependencies for all packages. In this case we can try using a Mariner package.

Reply:
I checked, and `python-tox` is in extended. Since this fix works as is, I'm going to merge it, but we should do something about `tox` and the issues it's caused across packages.